src/main/java/ro/msg/learning/shop/configuration/RouteMatrix.java

```diff
@@ -0,0 +1,14 @@
+package ro.msg.learning.shop.configuration;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@Data
+@ConfigurationProperties(prefix = "routematrix")
```

Comment:
Just as an FYI: usually it is common to have a "namespace" for custom application properties to avoid clashes with built-in Spring properties (e.g. to have all your custom property names starting with `ro.msg.shop.`)